src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -17,9 +17,9 @@
     public int matchResult(int result1, int result2, int bet1, int bet2) {
         return (bet1 == result1) & (result2 == bet2)
                 ? 2 :
-                (result1 > result2) & (bet1 > bet2)
-                        || (result1 < result2) & (bet1 < bet2)
-                        || (result1 == result2) & (bet1 == bet2)
+                (result1 > result2) && (bet1 > bet2)
+                        || (result1 < result2) && (bet1 < bet2)
+                        || (result1 == result2) && (bet1 == bet2)
                 ? 1 : 0;
```

Comment:
```suggestion
                        || (result1 == result2) && (bet1 == bet2) ? 1 : 0;
```
nice, one last thing: let's make code a bit shorter, and that would be it

Reply:
done